_posts/2016-08-19-CD-for-UX-and-PO.md

```diff
@@ -0,0 +1,38 @@
+
+“But doesn’t Agile solve this problem?” Not really. By simply doing your project in agile mode doesn’t give you the real insight in what should be done. You will have more flexibility with client’s demands, which however, won’t be based on actual information unless you are able to collect feedback from real usage. And even in that case, by using SCRUM, you can propably react to the feedback at earliest in the next sprint – which could take a month (or so).
+
+## Let clients and users know what you are doing
+
+We have been tackling these problems by combining Continuous Delivery (CD) with Kanban board and pilot user programs. Continuous Delivery allows us to deploy new versions of the product literally in minutes with a minimal effort. Forgetting SCRUM and formal sprints we use a constantly iterating Kanban board and have weekly meetings with our clients. This means we can prioritize things constantly and keep our focus on those that really matter to our client and to our users. 
+
+Continuous Delivery and Kanban are just tools to take a real advantage of a pilot program. There is nothing more valuable for the User Experience (UX) designer or for the Product Owner (PO) than user feedback from the real usage of the product. Any amount of workshopping or usability testing will not give you the same quality of data and feedback to focus on your work. 
+
+## Speed up for happy clients
+
+Later on when you go in full production mode, Continuous Delivery will allow you to react to the feedback quickly. For example, if we get a report about a bug or a usability problem around 9 am, we see it as a critical issue and we fix it right away, we can release a fixed version of the product already during client’s lunch break around 12 o’clock. With SCRUM and more cumbersome release setup you might take this up in the next sprint starting in two weeks, wait until the end of the sprint and then release it with a bunch of other bug fixes and features. Compare – 3 hours or 4 weeks. Guess which option your client values more?
+
+## Try to fail frequently
+
+What makes Continuous Delivery shine in this setup, is its habit of making failing cheap. You don’t need to spend that much time doing research on a feature as you have the option to sketch out basic functionality of the feature and try it out with your pilot users. Based on the feedback, you can iterate or scrap the feature.  
+
+All this means, you need to rethink the way of doing UX and PO work. You need to setup ways of collecting user data for example through analytics, feedback forms and interviews. You need to create and support a pilot user community and motivate it to use the unfinished product and to give feedback. In best case, the pilot community is a great resource for the product development and to market the product.
```

Comment:
"You need to setup ways of collecting user data for example" - ei väärin, mutta voisi olla ihan hyvin suoraviivaisemmin ilmaistu: "You need to collect user data, for example, "